tools/measurement/apksize/src/inappmessagingdisplay/inappmessaging-display.gradle

```diff
@@ -30,4 +30,7 @@ android {
 }
 dependencies {
     inappmessagingdisplayImplementation project(":firebase-inappmessaging-display")
+    inappmessagingdisplayImplementation ("com.google.firebase:firebase-inappmessaging:17.0.3") {
```

Comment:
Why do you want to remove the head version?

Reply:
It doesn't build otherwise, complaining it cannot find "firebase-common.aar"

```
* What went wrong:                                                                                                                                                                                                          
Execution failed for task ':tools:measurement:apksize:preInappmessagingdisplayAggressiveBuild'.
> Could not resolve all files for configuration ':tools:measurement:apksize:inappmessagingdisplayAggressiveRuntimeClasspath'.                                                                                               
   > Could not find firebase-common.aar (project :firebase-common). (
```